var/spack/repos/builtin/packages/squashfuse/package.py

```diff
@@ -19,6 +19,13 @@ class Squashfuse(AutotoolsPackage):
 
     version("master", branch="master")
     version("0.5.0", sha256="d7602c7a3b1d0512764547d27cb8cc99d1b21181e1c9819e76461ee96c2ab4d9")
+    version("0.4.0", sha256="646e31449b7914d2404933aea88f8d5f72c5d135d7deae3370ccb394c40d114a")
```

Comment:
I wonder if it’s worth commenting why we don’t have 0.3.0?

Reply:
I think it is OK to omit a comment; prior to this PR the package skipped over a number of versions without comment. Also, the Charliecloud PR will avoid 0.3.0 should it be added here in the future.